bin/ruby-build

```diff
@@ -342,6 +342,10 @@ fetch_tarball() {
   local package_filename="${package_name}.tar.gz"
 
   if [ "$package_url" != "${package_url%bz2}" ]; then
+    if [ -z "$(which bzip2)" ]; then
+      echo "bzip pacakage detected, however no bzip2 found within path" >&4
+      echo "If build fails, consider installing bzip2 pacakge" >&4
```

Comment:
There are some spelling errors in the warning message. Could we rewrite the warning message as just one line, maybe something like:

> warning: bzip2 not found; consider installing \`bzip2\` package